### DISCUSSION TITLE
Life.

### DISCUSSION THREAD

**Discussion by p0ryae** [Lazap-Development/lazap#42] [Off-topic]
Let's talk about life.

**Comment by LegItMate** [comment]
Deleting because discussion on a pointless topic.

**Reply by mister-coded** [reply]
stfu

**Reply by p0ryae** [reply]
OH NO ITS THAT KID THAT TELLS THE TEACHER ABOUT HW!!

**Reply by mister-coded** [reply]
> OH NO ITS THAT KID THAT TELLS THE TEACHER ABOUT HW!!

🤡

**Comment by p0ryae** [comment]
every morning I wake up and breath, eat, and drink lazap as my breakfast.

**Reply by mister-coded** [reply]
shit i wake up wondering where i am my hair looks like a pussy and my face looks like itss on drugs

**Reply by p0ryae** [reply]
bro what are you on....

**Reply by mister-coded** [reply]
idk pal i apologize

**Reply by voidedlegacy** [reply]
I wake up off 400mg of fucking caffeine

**Comment by mister-coded** [comment]
i have depression i listen to lil pepsi and world juice i feel no joy :sadpepe:

**Reply by JayyDoesDev** [reply]
:peepoglad: :thumbsup:

**Comment by blazklemencic** [comment]
lazap

**Comment by FadedFearSim** [comment]
cock

**Comment by voidedlegacy** [comment]
cock and melons.

**Comment by JayyDoesDev** [comment]
What's up guys! It's Quandale Dingle here! (RUUEHEHEHEHEHEEHE) I have been arrested for multiple crimes (AHHHHHHHHHHHHH) including: Battery on a police officer (WHAT), Grand theft, Declaring war on Italy, and public indecency (RUHEHEHEEHEHEHEHEHEHEHE X2 speed). I will be escaping prison on, MARCH 28TH! After that.... I WILL TAKE OVER THE WORLD

**Reply by voidedlegacy** [reply]
stfu.

**Reply by JayyDoesDev** [reply]
no

**Reply by voidedlegacy** [reply]
okay

**Comment by FadedFearSim** [comment]
cock

**Reply by ASOwnerYT** [reply]
Yes quite indeed I agree with your statement

**Comment by blazklemencic** [comment]
I found warm white liquid in my cereal this morning

**Comment by Assassin-1234** [comment]
I found cereal in my warm white liquid this morning

**Comment by NightMasterX** [comment]
I found cereal in my warm white liquid this morning

**Reply by voidedlegacy** [reply]
I warm liquid was really chunky.

**Comment by JayyDoesDev** [comment]
sir, please stop spamming my notifications sir. this isn't epic games sir

**Reply by blazklemencic** [reply]
Sounds like a skill issue

**Reply by Assassin-1234** [reply]
you can unsubscribe

**Reply by voidedlegacy** [reply]
no sounds like a your problem

**Reply by Assassin-1234** [reply]
> no sounds like a your problem

bend down, its time.. \*unzips\* \*takes pp out\* \*slams it on your face\* \*knocked unconscious\* \*gets umbrella\* 🌂🍑☂️💥

**Reply by blazklemencic** [reply]
*finds out it was his brother* *starts packing* *moves to alabama*

**Comment by JayyDoesDev** [comment]
@everyone

**Comment by JayyDoesDev** [comment]
Owo daily <@419958345487745035> <@751171434776232078> <@462071905185431563> <@753845579670487128>

**Comment by JayyDoesDev** [comment]
🥯⭐️🙏😝🥖🥐🗿🏁😡👩‍🦳🧢🥨🤓👋🟣🇯🇵😍🥪🧊😆🙁👍🍞🍒💀🥤🥹😐

**Comment by JayyDoesDev** [comment]
Greetings! Quandale Dingle here. My cousin Henry Dinglenut got arrested for putting TNT in a daycare center. (WHAT THE FUCK??) I put a camera in Joe Biden's bathroom and watched him take a poop. (WHAT????) My Asian brother, Quanliling Dingle put illegal substances in my ramen and I died.

**Comment by JayyDoesDev** [comment]
Hey, Quandale Dingle here. I just escaped prison and staying at Juandale Pringle's house. As I was running away from cops, I fell and scraped some of my foreskin off. A guy named Garfield Jenson bit me over in the shower while I was in prison. My baby momma Shiniqua Inderson told me to pay child support so I gave my baby to a creepy old guy,

**Comment by Uday-007** [comment]
I have finally recovered my account.
I would like to thank my momma without her this wouldn't be possible and also i would like to thank myself for believing in myself. Amen

**Comment by p0ryae** [comment]
This discussion is golden.

**Reply by JayyDoesDev** [reply]
Hey, Quandale Dingle here. I just escaped prison and staying at Juandale Pringle's house. As I was running away from cops, I fell and scraped some of my foreskin off. A guy named Garfield Jenson bit me over in the shower while I was in prison. My baby momma Shiniqua Inderson told me to pay child support so I gave my baby to a creepy old guy,

**Reply by ASOwnerYT** [reply]
Um actually, quadale dingle is a dead meme 🤓

**Reply by p0ryae** [reply]
for the love of god ENOUGH IS ENOUGH

**Reply by JayyDoesDev** [reply]
> Um actually, quadale dingle is a dead meme 🤓

sorry sir but you have yt in your name, mega ratio

**Comment by JayyDoesDev** [comment]
new video when

**Reply by p0ryae** [reply]
92.28.211.213

**Comment by Baggette** [comment]
dash upload yt vid challenge (improbable)

**Reply by p0ryae** [reply]
142.107.196.16

**Comment by Assassin-1234** [comment]
⠀⠀⠀⠀⠀⢰⡿⠋⠁⠀⠀⠈⠉⠙⠻⣷⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⢀⣿⠇⠀⢀⣴⣶⡾⠿⠿⠿⢿⣿⣦⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⣀⣀⣸⡿⠀⠀⢸⣿⣇⠀⠀⠀⠀⠀⠀⠙⣷⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⣾⡟⠛⣿⡇⠀⠀⢸⣿⣿⣷⣤⣤⣤⣤⣶⣶⣿⠇⠀⠀⠀⠀⠀⠀⠀⣀⠀⠀

**Reply by p0ryae** [reply]
Bro thought he could get away with it.

**Reply by Assassin-1234** [reply]
> Bro thought he could get away with it.

yes I did.

**Reply by LegItMate** [reply]
😳

**Reply by p0ryae** [reply]
No.

**Reply by Assassin-1234** [reply]
fine.

**Reply by Assassin-1234** [reply]
⠀⠀⠀⠀⠀⢀⣴⡾⠿⠿⠿⠿⢶⣦⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⢠⣿⠁⠀⠀⠀⣀⣀⣀⣈⣻⣷⡄⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⣾⡇⠀⠀⣾⣟⠛⠋⠉⠉⠙⠛⢷⣄⠀⠀⠀⠀⠀⠀⠀
⢀⣤⣴⣶⣿⠀⠀⢸⣿⣿⣧⠀⠀⠀⠀⢀⣀⢹⡆⠀⠀⠀⠀⠀⠀
⢸⡏⠀⢸⣿⠀⠀⠀⢿⣿⣿⣷⣶⣶⣿⣿⣿⣿⠃⠀⠀⠀⠀⠀⠀
⣼⡇⠀⢸⣿⠀⠀⠀⠈⠻⠿⣿⣿⠿⠿⠛⢻⡇⠀⠀⠀⠀⠀⠀⠀
⣿⡇⠀⢸⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣤⣼⣷⣶⣶⣶⣤⡀⠀⠀
⣿⡇⠀⢸⣿⠀⠀⠀⠀⠀⠀⣀⣴⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣦⡀

this one is better anyways

**Reply by p0ryae** [reply]
Bro where are you getting these from 💀

**Reply by Assassin-1234** [reply]
cant have shit in Detroit man.

**Reply by voidedlegacy** [reply]
This is very immature, Lazap-Development is for development on Lazap! Not such preposterous potty mouth!

**Reply by Baggette** [reply]
> This is very immature, Lazap-Development is for development on Lazap! Not such preposterous potty mouth!
 
kys (keep yourself safe)

**Reply by voidedlegacy** [reply]
> > This is very immature, Lazap-Development is for development on Lazap! Not such preposterous potty mouth!
> 
> kys (keep yourself safe)

Go suck a giant girthy slimy polly pecker you mother fricken bread man.

**Reply by Xtr126** [reply]
https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct

**Reply by voidedlegacy** [reply]
Who the fuck are you?

**Reply by Xtr126** [reply]
> Who the fuck are you?

Is that even important

**Reply by mister-coded** [reply]
it is indeed

**Reply by JayyDoesDev** [reply]
> > Who the fuck are you?
> 
> 
> 
> Is that even important 

yes because this entire thread knows each other

**Reply by Xtr126** [reply]
This is a public repository.

**Reply by voidedlegacy** [reply]
doesn't matter, just because it's "public" doesn't mean you have the right to join in on our argument.

**Reply by Xtr126** [reply]
I was doing my own thing. Not "join some argument"

**Reply by p0ryae** [reply]
> doesn't matter, just because it's "public" doesn't mean you have the right to join in on our argument.

This is a public repository and a public discussion. You have no right to tell people what to do. 

Closing this discussion it's getting out of hand.
